js/timetable.js

```diff
@@ -479,7 +479,10 @@ var Course = {
             // Add course style class.
             for (var i = 1; i <= 6; i++) {
                 if (i === 6 || $('.lesson-style-' + i).length < 1) {
-                    $('.lesson[data-name=' + courseName + ']').parent().addClass('lesson-style-' + i);
+                     //https://stackoverflow.com/questions/6469993/jquery-selector-problem-when-using-a-single-parenthesis
+                    var escapedcourseName = courseName.replace('(', '\\(').replace(')', '\\)')
```

Comment:
might just be the github syntax highlighter but this is kind of hard to read, can you put the replaces on new lines? other than that all good